Code/src/main/java/nl/utwente/viskell/ui/components/BinderAnchor.java

```diff
@@ -0,0 +1,42 @@
+package nl.utwente.viskell.ui.components;
+
+import nl.utwente.viskell.haskell.expr.Binder;
+import nl.utwente.viskell.haskell.expr.LetExpression;
+
+// TODO make this an independent class if other blocks (case?) need this too 
+/** An internal output anchor for an argument binder. */
+public class BinderAnchor extends OutputAnchor {
+
+    /**
+     * 
+     */
+    private final LambdaContainer lambdaContainer;
+
+    // FIXME BinderAnchor should not have or use the DefinitionBlock parent
```

Comment:
FIXME seems to be outdated (same for duplicated comment in `ResultAnchor` class.)